src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -29,6 +34,9 @@ public CommandBox(CommandExecutor commandExecutor) {
         this.commandExecutor = commandExecutor;
         // calls #setStyleToDefault() whenever there is a change to the text of the command box.
```

Comment:
If I'm not wrong the # symbol is placed after a class name to show that the method following the # belongs to a class. Perhaps it might be clearer to do so such that other developers will know where to find this method?

Reply:
That's a comment from the code base, no idea.